docs/API/InstrumentFormat.md

```diff
@@ -287,32 +342,188 @@ A NumericElement represents a numeric data input and has the general form of:
 
 
 
-### 2.1.5: ScoreFieldElement
+### 2.1.5: CalcFieldElement
 
-A score field represents a placeholder to display/save values based on other
+A calc field represents a placeholder to display/save values based on other
 data entered by the user in the instrument but does not directly get input from
-the user. It has the following form.
+the user. It is displayed as a disabled text box. It has the following form.
 
 ```js
 {
-    "Type": "score",
+    "Type": "calc",
     "Name": REQUIRED,
     "Description": OPTIONAL,
+    "Formula" : string, /* LORIS Logic Parser formula */
     "Options": {
         /* None currently */
     }
 }
 ```
 
-`Type`: MUST be "score".
+`Type`: MUST be "calc".
 
-`Name`: Required. Follows `PageElement.Name` rules. The Name MAY be used by an
+`Name`: Required. Follows `PageElement.Name` rules. The Name will be used by an
         implementation as a field name to save calculated data into.
 
-`Description`: Optional. Follows `PageElement.Name` rules. If not specified, the
-               score will be displayed with no accompagning text.
+`Description`: Optional. If not specified, the
+               score will be displayed with no accompanying text.
```

Comment:
Why was the sentence describing the format of the field removed?

Reply:
Does it follow `PageElement.Name` rules or `PageElement.Description` rules?